src/types/TodoStatus.ts

```diff
@@ -0,0 +1,5 @@
+export enum TodoStatus {
+  all = 'All',
+  active = 'Active',
+  completed = 'Completed',
+}
```

Comment:
Don't need to duplicate enums with the same values, use only one enum 
<img width="620" alt="image" src="https://github.com/user-attachments/assets/5816902b-7437-418c-adbd-884f1447854e">